packages/isle-of-man/README.md

```diff
@@ -0,0 +1,27 @@
+# Isle of Man
```

Comment:
Could you add the flag 🇮🇲 and a link in the [main README](https://github.com/VictorCazanave/svg-maps/blob/master/README.md) please?